collectors/0/goproc.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import traceback
+
+import requests
+import sys
+import time
+
+from collectors.etc import goprocconf
```

Comment:
We are trying to move away from py based conf to yml based conf. Could you inline the yml reading in this collector?

Reply:
Done